Repo-Integration/docker-compose.yaml

```diff
@@ -20,7 +20,7 @@ services:
 
   app:
     build: 
-      context: ${BASE_DIR}/latest/wss-${SCM}-app/docker
+      context: ${BASE_DIR}/latest/wss-gls-app/docker
```

Comment:
You replaced the '${SCM}' variable with the hard-coded "gls", but I'm assuming this example is applicable to the other repo integrations as well, isn't it?
If so, consider keeping the SCM variable (and perhaps enhance this example by actually declaring that variable in the pipeline, because I believe it's not in the other example).